src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -1380,7 +1380,7 @@ public void setDisplayCardValues() {
                 result.setImageUrl(thumbnailServiceWrapper.getDataverseCardImageAsBase64Url(result));
             } else if (result.getType().equals("datasets")) {
                 if (result.getEntity() != null) {
-                    result.setImageUrl(thumbnailServiceWrapper.getDatasetCardImageAsBase64Url(result));
+                    result.setImageUrl(thumbnailServiceWrapper.getDatasetCardImageAsUrl(result));
```

Comment:
I'm not sure why I'm seeing a broken thumbnail on the search card:

![Screenshot 2023-12-04 at 3 49 10 PM](https://github.com/IQSS/dataverse/assets/21006/f0deee2e-b595-4363-a735-ba1978536ca8)

Nothing in server.log. Works fine on demo. Here are the two files I uploaded:

![9568-homer](https://github.com/IQSS/dataverse/assets/21006/d35bd82c-6d4b-4d0f-8577-abb4dbf4907f)
[Architecture ideas - Sheet1.pdf](https://github.com/IQSS/dataverse/files/13551225/Architecture.ideas.-.Sheet1.pdf)

In short, if you set PNG (of Homer) as the dataset thumbnail, it looks weird, as above. A workaround is to remove it so that people see the default dataset icon.

Reply:
Hmm, just tested at QDR, which has the original version of the code and it works OK as far as I can tell: 
![image](https://github.com/IQSS/dataverse/assets/6731983/5dd68bfe-7fbe-4956-a221-ab5d7cc00bd6)
. I can look to see if there are code differences and try to set up this branch somewhere. (FWIW: I can't get your exact homer image - github says "This private-user-images.githubusercontent.com page can’t be found" when I try to copy it/it's link.

Reply:
The pdf works OK too.

Reply:
@qqmyers I'm sorry but QDR is a fork and not a good test. I just switched to develop and thumbnails work fine. Then I switched back to this branch and I'm getting 404s:

![Screenshot 2023-12-05 at 9 06 14 AM](https://github.com/IQSS/dataverse/assets/21006/0eae9cfa-d81f-4abd-a9a6-1b6fb4c14ffe)

Let's pick which server to test on (I've been testing on Docker on my laptop). Either dataverse-internal or a newly spun up EC2 instance. I'll reach out in Slack.

Reply:
This seems to be fixed as of 7349ed9f75.